packages/gatsby/content/advanced/migration.md

```diff
@@ -37,9 +37,26 @@ Run `npx @yarnpkg/doctor .` (or `yarn dlx @yarnpkg/doctor .`) in your project to
 
 Note that the doctor is intended to report any potential issue - it's then up to you to decide whether they are a false positive or not (for example it won't traverse Git repositories). For this reason we don't recommend using it as a CI tool.
 
-### Use `yarn dlx` instead of `yarn global`
-
-`yarn dlx` is designed to execute one off scripts that may have been installed as global packages with `yarn 1.x`. Managing system-wide packages is outside of the scope of `yarn`. To reflect this, `yarn global` has been removed. [Read more on GitHub](https://github.com/yarnpkg/berry/issues/821).
```

Comment:
🤔 I think this information should be added back as a note to the `yarn global` entry.

Reply:
Yep, it's important enough that it's worth keeping this section, and make the "see more" link point there.

Reply:
Added back below the changes table, and a read more link in the table pointing to the section.